static/app/views/issueList/groupSearchViewTabs/draggableTabBar.tsx

```diff
@@ -1,359 +0,0 @@
-      onRename: () => setEditingTabKey(tab.key),
-      onDuplicate: () => handleDuplicateView(tab),
-      onDelete: tabs.length > 1 ? () => handleDeleteView(tab) : undefined,
-    });
-  };
-
-  const allTabs = tempTab ? [...tabs, tempTab] : tabs;
-
-  return (
-    <DraggableTabList
-      onReorder={(newOrder: Node<DraggableTabListItemProps>[]) =>
-        dispatch({
-          type: 'REORDER_TABS',
-          newKeyOrder: newOrder.map(node => node.key.toString()),
-        })
-      }
-      onReorderComplete={() => dispatch({type: 'SYNC_VIEWS_TO_BACKEND'})}
-      defaultSelectedKey={initialTabKey}
-      onAddView={handleAddView}
-      orientation="horizontal"
-      editingTabKey={editingTabKey ?? undefined}
-      hideBorder
-    >
-      {allTabs.map(tab => (
-        <DraggableTabList.Item
-          textValue={tab.label}
-          key={tab.key}
-          to={normalizeUrl({
-            query: {
-              ...queryParams,
-              query: tab.unsavedChanges?.[0] ?? tab.query,
-              sort: tab.unsavedChanges?.[1] ?? tab.querySort,
-              viewId: tab.id !== TEMPORARY_TAB_KEY ? tab.id : undefined,
-            },
-            pathname: `/organizations/${organization.slug}/issues/`,
-          })}
-          disabled={tab.key === editingTabKey}
-        >
-          <TabContentWrap>
-            <EditableTabTitle
-              label={tab.label}
-              isEditing={editingTabKey === tab.key}
-              setIsEditing={isEditing => setEditingTabKey(isEditing ? tab.key : null)}
-              onChange={newLabel =>
-                dispatch({type: 'RENAME_TAB', newLabel: newLabel.trim(), syncViews: true})
-              }
-              isSelected={
-                (tabListState && tabListState?.selectedKey === tab.key) ||
-                (!tabListState && tab.key === initialTabKey)
-              }
-            />
-            {/* If tablistState isn't initialized, we want to load the elipsis menu
-                for the initial tab, that way it won't load in a second later
-                and cause the tabs to shift and animate on load.
-            */}
-            {((tabListState && tabListState?.selectedKey === tab.key) ||
-              (!tabListState && tab.key === initialTabKey)) && (
-              <motion.div
-                // This stops the ellipsis menu from animating in on load (when tabListState isn't initialized yet),
-                // but enables the animation later on when switching tabs
-                initial={tabListState ? {opacity: 0} : false}
-                animate={{opacity: 1}}
-                transition={{delay: 0.1, duration: 0.1}}
-              >
-                <DraggableTabMenuButton
-                  hasUnsavedChanges={!!tab.unsavedChanges}
-                  menuOptions={makeMenuOptions(tab)}
-                  aria-label={t(`%s Ellipsis Menu`, tab.label)}
-                />
-              </motion.div>
-            )}
-          </TabContentWrap>
```

Comment:
Moved to its own component `IssueViewTab.tsx` 

This contains the tab's content like the editable title and ellipsis menu